dashboard/app/models/follower.rb

```diff
@@ -41,4 +41,9 @@ def section_must_belong_to_teacher
   validate :cannot_follow_yourself, :teacher_must_be_teacher, :section_must_belong_to_teacher
 
   validates_presence_of :user, :student_user, :section
+
+  after_create :assign_script
+  def assign_script
+    student_user.assign_script(section.script) if section.try(:script)
```

Comment:
yes, the UI allows you to create a section without choosing a script.

irb(main):001:0> Section.where(script_id: nil).count
=> 62322